whois-update/src/main/java/net/ripe/db/whois/update/mail/MailGatewaySmtp.java

```diff
@@ -108,4 +109,11 @@ private void sendEmailAttempt(final String to, final String replyTo, final Strin
             }
         }
     }
+
+    private void setHeaders(final MimeMessage mimeMessage, final String messageId) throws MessagingException {
```

Comment:
Do we need a separate method?

Reply:
I wanted to keep it separated. We don't need it but it is more clear at least for me. I prefer this rather than having a big method (sendEmailAttempt)
anyways this task is not ready yet, I need to create the messageId and I will do this in the other task about the bounce detection

Reply:
OK